confetti/src/main/java/com/github/jinatonic/confetti/confetto/BitmapConfetto.java

```diff
@@ -42,8 +42,7 @@ public int getHeight() {
     }
 
     @Override
-    protected void drawInternal(Canvas canvas, Matrix matrix, Paint paint, float x, float y,
-            float rotation, float percentageAnimated) {
+    protected void drawInternal(Canvas canvas, Matrix matrix, Paint paint, float x, float y, float vX, float vY, float rotation, float vRotation, float percentageAnimated) {
```

Comment:
I would make the naming scheme a little more descriptive, e.g. xVelocity rather than vX.